libs/payments/paypal/src/lib/paypal.manager.spec.ts

```diff
@@ -134,15 +113,15 @@ describe('PayPalManager', () => {
       const token = faker.string.uuid();
       const mockNewBillingAgreement = NVPBAUpdateTransactionResponseFactory();
 
-      paypalCustomerManager.fetchPaypalCustomersByUid = jest
-        .fn()
-        .mockResolvedValueOnce([]);
+      jest
+        .spyOn(paypalCustomerManager, 'fetchPaypalCustomersByUid')
+        .mockResolvedValue([]);
 
-      paypalClient.createBillingAgreement = jest
-        .fn()
-        .mockResolvedValueOnce(mockNewBillingAgreement);
+      jest
+        .spyOn(paypalClient, 'createBillingAgreement')
+        .mockResolvedValue(mockNewBillingAgreement);
 
-      expect(
+      await expect(
         paypalManager.getOrCreateBillingAgreementId(uid, true, token)
       ).rejects.toBeInstanceOf(PaypalManagerError);
       expect(paypalClient.createBillingAgreement).not.toBeCalled();
```

Comment:
Q: is there a preference between using `.toBeCalled()` or `.toHaveBeenCalled()`?

Reply:
They're aliases of eachother so no preference.

Reply:
Pretty even split. 🤷 
```sh
git grep -Pho "\.(toBeCalled|toHaveBeenCalled)\(" libs apps | sort -i | uniq -c | sort -nr
  10 .toBeCalled(
   8 .toHaveBeenCalled(
```